odoo/addons/base/models/res_users_view.py

```diff
@@ -19,19 +48,41 @@ def _get_view_group_hierarchy(self):
                 'name': section.name,
                 'categories': [
                     {
-                        'id': category.id,
-                        'name': category.name,
-                        'description': category.description,
-                        'groups': [[group.id, group.name] for group in category.group_ids]
-                    } for category in section.child_ids.sorted(lambda c: c.sequence) if category.group_ids
+                        'id': privilege.id,
+                        'name': privilege.name,
+                        'description': privilege.description,
+                        'placeholder': privilege.placeholder,
+                        'groups': [[group.id, group.name] for group in privilege.group_ids]
+                    } for privilege in section.privilege_ids.sorted(lambda c: c.sequence) if privilege.group_ids
```

Comment:
Why do you explicitly sorted here ? Since `privilege_ids` is already order by sequence, it shouldn't be necessary (except if it to cover the case or the ORM doesn't sorted X2many when it updates the inverse field ? :thinking: )